### PR TITLE
feat(web): Organization published material page

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Organization published material page

## What

* Added a page that organization's can utilize for their published material (publicly accessible pdf, excel, word files and so forth)
* Added a content type in Contentful called EnhancedAsset
* Added a content type in Contentful called GenericTagGroup
* Added a fail-safe to the NewstList component in case the slug is not found (I came across a 500 error due to this)

## Why

* Organization's needed a way to show their published material on the island.is web

## Screenshots / Gifs

https://user-images.githubusercontent.com/43557895/161961635-4b068636-56c6-4665-8c47-8912bccd0555.mp4

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import { Box, FocusableBox, Text, Tag } from '@island.is/island-ui/core'
+import { EnhancedAsset } from '@island.is/web/graphql/schema'
+import { useDateUtils } from '@island.is/web/i18n/useDateUtils'
+
+const getFileEnding = (url: string): string => {
+  const lastChunk = url.split('/').pop()
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (kksteini) [inline apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx:8]: This eslint directive doesn't look necessary. Is this acting weird?

Comment (kksteini) [inline apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx:8]: This eslint directive doesn't look necessary. Is this acting weird?

**Review by kksteini — APPROVED** [review]
LGTM

File: apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/screens/Organization/PublishedMaterial/components/PublishedMaterialItem/PublishedMaterialItem.tsx:8]: This is something that I forgot, that function used to be different and I changed it at some point.

**Review by siggifv — APPROVED** [review]

**Pull request merged**